packages/connect/src/SmoldotProvider/SmoldotProvider.ts

```diff
@@ -8,7 +8,7 @@ import {
 } from '@polkadot/rpc-provider/types';
 import { assert, logger } from '@polkadot/util';
 import EventEmitter from 'eventemitter3';
-import * as smoldot from 'smoldot';
+import * as smoldot from '@substrate/smoldot-light';
```

Comment:
NOt for right now, but it would be good to stop doing this `import * as smoldot...` everywhere so that we don't have all this repetetive `smoldot.SmoldotChain` (etc) usages